<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionWindows</class>
 <widget class="QWidget" name="SelectionWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelectionWindows</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="l_serialSelection">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Seleziona la porta seriale desiderata            </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_SerialList"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="m_textBox"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="l_selectionButton">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_quitButton">
        <property name="text">
         <string>Esci</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_selectionButton">
        <property name="text">
         <string>Seleziona</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
